source/fundamentals/crud/write-operations.txt

```diff
@@ -2,7 +2,8 @@
 Write Operations
 ================
 
-.. default-domain:: mongodb
+.. meta::
+   :description: Learn the concepts and commands for MongoDB write operations by using the {+driver-long+}.
```

Comment:
S:
```suggestion
   :description: Learn the concepts and commands for executing MongoDB write operations by using the {+driver-long+}.
```

Reply:
I considered "running" while formulating the sentence. However, parallelism can be tricky. While "executing" agrees with "commands", it doesn't agree with "concepts". And this form: "Learn the concepts of and commands for executing" unfortunately isn't proper grammar either.
Maybe "Learn about the commands for running ..." might be a good tradeoff. What do you think?